Source/WebCore/html/HTMLElement.h

```diff
@@ -54,6 +54,7 @@ enum class PopoverState : uint8_t {
     Auto,
     Manual,
 };
+enum class ToggleState : bool;
```

Comment:
The forward declarations should not be mixed in with actual definitions. `FireEvenfs`, `FocusPreviousElement`, and `PopoverState` should be moved out of this paragraph and below all the forward declarations.